### PR TITLE
fix #32822 zmz2019.com

### LINKED ISSUE #32822: zmz2019.com
### Issue URL (Ads)
[http://www.zmz2019.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.zmz2019.com%2F)
### Comment
Username: @zxysm
> 每个页面稍作停留就会出现画面正中间的广告，并且整个页面变暗。

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.13
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Block WebRTC, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>EasyList China,<br>Fanboy's Annoyances,<br>AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard French,<br>AdGuard Base,<br>Adblock Warning Removal List,<br>(Obsolete) Anti-Adblock Killer | Reek,<br>ABPVN List,<br>List-KR,<br>Fanboy's Cookiemonster List,<br>AdGuard Tracking Protection,<br>AdGuard German,<br>AdGuard Japanese,<br>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/32822

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -788,6 +788,14 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26330
 ||acs.youku.com/*.youku.play.$replace=/("ad":{).+"(\}\,"playlog")/\$1\$2/
 @@||cibntv.net/youku/*.mp4?$domain=v.youku.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/32822
+zmz2019.com##.key-result > li > a[href*=".taobao.com"]
+||jstucdn.com/g3/
+.com/js/ad_$domain=zmz2019.com
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: I suggest this rule as generic rule (EasyList China has it already as generic rule).
What do you think @Alex-302?

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -788,6 +788,14 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
...
+||nqtnu.cn/V.aspx?Bln=$domain=zmz2019.com
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: EasyList China already his this rule, maybe apply it to our filter?

/V.aspx?Bln=$script,third-party


Or maybe we should fork EasyList China and make our own Chinese filter? Currently we have 732 rules for Chinese (comments counted.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Some rules from EasyList China - they block ads good and on this site i didn't notice something broken.
For this PR I limited to domain cause idk should we add this rules to our filters (in future maybe our own Chinese):

`|http://ad.`
`.com/js/ad_`
`/V.aspx?Bln=$script,third-party`

**Comment by zloyden** [comment]
Some rules from EasyList China - they block ads good and on this site i didn't notice something broken.
For this PR I limited to domain cause idk should we add this rules to our filters (in future maybe our own Chinese):

`|http://ad.`
`.com/js/ad_`
`/V.aspx?Bln=$script,third-party`

**Comment by Alex-302** [comment]
Please always check with user's filters. Don't make duplicates with foreign filters(excluding Safari, because filters are simplified).

**Comment by zloyden** [comment]
I always check rules in UF before add. EasyList China is enough for this site.


> Don't make duplicates with foreign filters(excluding Safari, because filters are simplified).

Ok :)

**Pull request closed without merging**